en/flight_modes/altitude.md

```diff
@@ -0,0 +1,55 @@
+# ALTITUDE CONTROL Flight Mode
+
+The ALTITUDE CONTROL flight mode maintains constant altitude when all remote control inputs are centered (i.e. no roll, pitch, yaw and at 50% throttle). The mode does not use GPS, and hence will not attempt to hold position against wind.
+
+> **Tip** This mode is recommended for beginners because it is easy to return the vehicle to predictable flight.
+
+<span></span>
+> **Note**
+>  * This mode is *assisted*. The autopilot makes it easier for users to control the vehicle. 
+>  * This mode requires that a radio control is connected.
+>  * The altitude is normally measured using an altitude barometer, which may become inaccurate in extreme weather conditions. Vehicles that include a LIDAR/range sensor will be able to control altitude with greater reliability and accuracy. 
+<!-- ? * The vehicle must be armed before this mode can be engaged. -->

```

Comment:
Thanks very much. I'll modify the line "This mode requires that a radio control is connected." appropriately.